ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/fips/provision/cucushift-installer-rehearse-aws-ipi-ovn-fips-provision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-fips-provision
+  steps:
+  - chain: ipi-conf-aws
+  - ref: ovn-conf
+  - ref: ingress-aws-nlb-manifest
```

Comment:
Per the chain name, it enables fips bases on `aws-ipi-ovn`, while `ingress-aws-nlb-manifest` indicate it based on `aws-ipi-ingress-nlb`